src/test/java/io/vertx/core/logging/JULLogDelegateTest.java

```diff
@@ -40,10 +41,24 @@
   public static void initialize() throws IOException {
     // Clear value.
     System.clearProperty("vertx.logger-delegate-factory-class-name");
+    LoggerFactory.initialise();
```

Comment:
@cescoffier this was missing, compared to other logging tests. Maybe that would solve the CI issue you were referring to in the class Javadoc? If so then we can remove the ignore annotations.